_episodes/12-cluster.md

```diff
@@ -82,65 +58,99 @@ permissions for on the cluster.
+notice that the current hostname is also part of our prompt!)
+
+## Nodes
+
+Individual computers that compose a cluster are typically called *nodes* (although you will also
+hear people call them *servers*, *computers* and *machines*). On a cluster, there are different
+types of nodes for different types of tasks. The node where you are right now is called the *head
+node*, *login node* or *submit node*. A login node serves as an access point to the cluster. As a
+gateway, it is well suited for uploading and downloading files, setting up software, and running
+quick tests. It should never be used for doing actual work.
+
+The real work on a cluster gets done by the *worker* (or *compute*) *nodes*. Worker nodes come in
+many shapes and sizes, but generally are dedicated to doing all of the heavy lifting that needs
+doing.
+
+All interaction with the worker nodes is handled by a specialized piece of software called a
+scheduler (the scheduler used in this lesson is called {{ site.workshop_shed_name }}). We'll
+learn more about how to use the scheduler to submit jobs next, but for now, it can also tell us
+more information about the worker
+nodes.
+
+For example, we can view all of the worker nodes with the `{{ site.sched_info }}` command.
+
+```
+{{ site.host_prompt}} {{ site.sched_info }}
+```
+{: .bash}
+```
+{% include /snippets/12/info.snip %}
+```
 {: .output}
 
-The admin explains that Lola has to work with this remote shell session in order to
-run programs on the cluster. Launching programs that open a Graphical User Interface
-(GUI) is possible, but the interaction with the GUI will be slow as everything will
-have to get transferred through the WiFi network her laptop is currently logged into.
-Before Rob continues, he suggests to leave the cluster node again. For this, Lola can
-type in `logout` or `exit`.
+There are also specialized machines used for managing disk storage, user authentication, and other
+infrastructure-related tasks. Although we do not typically logon to or interact with these machines
+directly, they enable a number of key features like ensuring our user account and files are
+available throughout the HPC system.
 
-~~~ 
-$ logout
-~~~
-{: .language-bash}
+**This is an important point to remember: files saved on one node (computer) are often available
+everywhere on the cluster!**
 
-## Looking around more
+## What's in a node? 
 
-The admin continues to encourage Lola to look around. She explains that all of a 
-cluster's nodes have similar components as Lola's laptop or workstation. 
+All of a HPC system's nodes have the same components as your own laptop or desktop: *CPUs* (sometimes
+also called *processors* or *cores*), *memory* (or *RAM*), and *disk* space. CPUs are a computer's
+tool for actually running programs and calculations. Information about a current task is stored in
+the computer's memory. Disk is a computer's long-term storage for information it will need later.
```

Comment:
Maybe reword: `Disk is a computer's long-term storage for information it will need later.`
A suggestion: `Disk refers to all storage that can be accessed like a filesystem. This is generally storage that can hold data permanently, i.e. data is still there even if the computer has been restarted.`

Reply:
Addressed in https://github.com/hpc-carpentry/hpc-intro/pull/101/commits/8e2ca7059afd9b0080cab714be51e85c712790cf